uhabits-android/src/main/java/org/isoron/uhabits/intents/PendingIntentFactory.kt

```diff
@@ -54,6 +54,15 @@ class PendingIntentFactory
                     },
                     FLAG_UPDATE_CURRENT)
 
+    fun cancelNotification(habit: Habit): PendingIntent =
```

Comment:
It is not clear what's the difference between `cancelNotification` and `dismissNotification`. I think it's better to name this `removeRepetition`.